src/sentry/snuba/discover.py

```diff
@@ -576,4 +589,27 @@ def get_facets(query, params, limit=10, referrer=None):
             ]
         )
 
+    if aggregate_tags:
```

Comment:
It only impacts environment. Individual tags and aggregate tags are not interleaved. The individual ones are the most popular and aggregate_tags are the least. (except for environment), since both lists are built in order according to top_tags.